tests/integration/targets/azure_rm_account_info/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+
+- name: Get facts for current logged in user
+  azure.azcollection.azure_rm_account_info:
+  register: result
+
+- assert:
+    that:
+      - result is not changed
+      - result is not failed
```

Comment:
It should be good to test that the result contains ``account_info`` and the all expected key from account_info, ``tenantId``, ``state``, etc

Reply:
Added.